src/lib.rs

```diff
@@ -561,6 +524,53 @@ impl<'text> BidiInfo<'text> {
     }
 }
 
+
+/// Contains a reference of `Bidi_info` and one of its `paragraphs`.
```

Comment:
Nit: I think we need more docs here.

Reply:
Do you mean add a usage? or illustrate more?

Reply:
Just illustrate more what this is for and why it exists.

Reply:
done